interface-definitions/service_dns_forwarding.xml.in

```diff
@@ -793,6 +793,184 @@
+                            <properties>
+                              <help>Periodic zone retrieval interval</help>
+                              <valueHelp>
+                                <format>&lt;minutes&gt;</format>
+                                <description>Retrieval interval in minutes</description>
+                              </valueHelp>
+                              <valueHelp>
+                                <format>&lt;minutes&gt;m</format>
+                                <description>Retrieval interval in minutes</description>
+                              </valueHelp>
+                              <valueHelp>
+                                <format>&lt;hours&gt;h</format>
+                                <description>Retrieval interval in hours</description>
+                              </valueHelp>
+                              <valueHelp>
+                                <format>&lt;days&gt;d</format>
+                                <description>Retrieval interval in days</description>
+                              </valueHelp>
+                              <constraint>
+                                <regex>[1-9]([0-9]*)([mhd]{0,1})</regex>
+                              </constraint>
+                            </properties>
+                            <defaultValue>24h</defaultValue>
+                          </leafNode>
+                        </children>
+                      </node>
+                      <leafNode name="retry-interval">
+                        <properties>
+                          <help>Retry interval after zone retrieval errors</help>
+                          <valueHelp>
+                            <format>u32:1-2147483647</format>
+                            <description>Retry period in seconds</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 1-2147483647"/>
+                          </constraint>
+                        </properties>
+                        <defaultValue>60</defaultValue>
+                      </leafNode>
+                      <leafNode name="max-zone-size">
+                        <properties>
+                          <help>Maximum zone size in megabytes</help>
+                          <valueHelp>
+                            <format>u32:0-1024</format>
+                            <description>Size in megabytes</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 0-1024"/>
+                          </constraint>
+                        </properties>
+                        <defaultValue>0</defaultValue>
+                      </leafNode>
+                      <leafNode name="zonemd">
+                        <properties>
+                          <help>ZONEMD mode</help>
```

Comment:
What's ZONEMD mode? Maybe add a bit more context. to the help string.


Please also add
```xml
<completionHelp>
  <list>ignore validate require</list>
</completionHelp>
```

to improve the user experience on the CLI

Reply:
fixed